documentation/asciidoc/topics/proc_installing_command_line.adoc

```diff
@@ -0,0 +1,66 @@
+[id='installing-openshift-command-line_{context}']
+= Installing the Helm chart on the command line
+
+[role="_abstract"]
+Use the `helm` and `oc` clients to install the {brandname} Helm chart from the Red Hat Helm Charts repository via the command line.
```

Comment:
This feels very wordy with "Infinispan Helm Chart".. could we just have "to install the chart from the Red Hat ... "